tests/test_xgboost.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import numpy as np
+import xgboost as xgb
```

Comment:
You'll need to add xgboost to requirements_dev.txt

Reply:
makes sense